dynawo/sources/ModelicaCompiler/DYNCompileModelicaModel.cpp

```diff
@@ -327,8 +343,8 @@ compileModelicaToC(const string& modelName, const string& fileToCompile, const v
   // add Modelica transcription
   mosFile << "// Translate model from Modelica to C/C++" << std::endl;
   mosFile << "setCommandLineOptions(\"" << runOptions() << "\");" << std::endl;
-  mosFile << "instantiateModel(" << modelName << "); getErrorString();" << std::endl;
-  mosFile << "translateModel(" << modelName << "); getErrorString();" << std::endl;
+  mosFile << "instantiateModel(" << packageName << modelName << "); getErrorString();" << std::endl;
```

Comment:
We force the user to add a '.' at the end of the package name here. Maybe we could add it automatically if missing?

Reply:
I will do it